client/layout/masterbar/style.scss

```diff
@@ -17,6 +16,7 @@ $autobar-height: 20px;
 	z-index: z-index( 'root', '.masterbar' );
 	-webkit-font-smoothing: subpixel-antialiased;
 	transition: transform 0.2s ease-out;
+	box-sizing: border-box;
```

Comment:
I'm worried about the wider impact of this change. `box-sizing` has the potential to affect styles that apply to the horizontal margins. If folks have relied on this property for some time then there might be unintended consequences of making this switch.

Personally I'd try and limit the scope of this change to just making a 1:1 swap between the Sass and CSS vars and avoid amending this property.

Reply:
> making a 1:1 swap between the Sass and CSS vars

In that case we would need to replace all `var(--masterbar-height)` with `calc( var(--masterbar-height) + 1px)` where 1px is the border height. Thus introducing some technical debt.

The new masterbar (for the nav-unification) already [didn't use `border` but `box-shadow` instead](https://github.com/Automattic/wp-calypso/pull/46035/files#diff-24092f405ff9cb89da5651239314443fL33-L37) which I changed to align better with the current code. That means that if we are not introducing `box-sizing: border-box;` we will inevitably may have  _unintended consequences_ once the nav-unification goes live.

Since the diff is only 1px, I would prefer avoiding some technical debt and address any _unintended consequences_ as soon as possible.

Reply:
You seem pretty confident and to be fair I cannot see any problems on testing so I'm inclined to agree with you. Nonetheless, let's continue to be aware of the potential for knock-on effects. Hopefully, they will be minimal 🤞 .